GettingStarted/src/Samples/DrawBuilding/SGSDrawBuildingVC.m

```diff
@@ -0,0 +1,69 @@
+//
+//  SGSDrawBuildingVC.m
+//  GettingStarted
+//
+//  Created by Fernando Sanchez Vilas on 22/05/2019.
+//  Copyright © 2019 Situm Technologies S.L. All rights reserved.
+//
+
+#import "SGSDrawBuildingVC.h"
+@import GoogleMaps;
+
+@interface SGSDrawBuildingVC ()
+@property (nonatomic,strong) GMSMapView *googleMapView;
+@end
+
+static NSString *NoFloorsInBuildingAlertTitle = @"No Floors";
+static NSString *NoFloorsInBuildingAlertBody = @"At least one floor is needed to draw the building.";
+static NSString *NoFloorsInBuildingOk = @"Ok";
+
+static NSString *DrawBuildingButtonText = @"Draw Building";
+
+@implementation SGSDrawBuildingVC
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    [self drawGoogleMap];
+    [self addDrawBuildingButton];
+}
+
+#pragma mark - Draw Google Map in Proper View
+-(void)drawGoogleMap{
+    GMSCameraPosition *camera = [GMSCameraPosition cameraWithLatitude:-33.86 longitude:151.20 zoom:6];
+    self.googleMapView = [GMSMapView mapWithFrame:CGRectZero camera:camera];
+    self.view=self.googleMapView;
+}
+
+-(void)addDrawBuildingButton{
+    UIBarButtonItem *drawBuildingButton = [[UIBarButtonItem alloc] initWithTitle:DrawBuildingButtonText style:UIBarButtonItemStylePlain target:self
+                                                                          action:@selector(drawBuilding)];
+    self.navigationItem.rightBarButtonItem=drawBuildingButton;
+}
+
+#pragma mark - Draw Building in Map
+- (void)drawBuilding{
+    //Move camera to building coordinates and set the desired zoom
+    GMSCameraPosition *cameraPosition = [GMSCameraPosition cameraWithTarget:self.selectedBuildingInfo.building.center zoom:19];
```

Comment:
Igual compensa mover estas dúas liñas a unha función separada? Así penso que quedaría máis claro que parte é relativa ao posicionamento da cámara e cal é cal é exclusivo do pintado do plano. Que opinas?

Reply:
Hmm, o que se pode facer entón e cambiar o nome do método e chamarlle show building e dividilo en dous, moveCamera e drawBuilding, pero penso que deberían estar no mesmo método, xa que se non se move a camara o usuario non o vai ver na pantalla e pode despistar se so copia o método de de pintalo. Como o ves?

Reply:
Perfecto 👍